### PR TITLE
Fix: Move some jvsctl commands to token subcommand

### PR DESCRIPTION
**Pull request by dandans-verily**
Move create and validate to token subcommand.
Fix: #246

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]

File: docs/cli.md [hunk 1]
```diff
@@ -54,5 +54,5 @@ If you installed JVS using the provided Terraform module as described in the
 authentication. E.g.
 
 ```sh
-jvsctl token --auth-token $(gcloud auth print-identity-token) -e "just testing"
+jvsctl token validate --auth-token $(gcloud auth print-identity-token) -e "just testing"
```
Comment (yolocs) [inline docs/cli.md:57]: This _was_ a token create command.
```suggestion
jvsctl token create --auth-token $(gcloud auth print-identity-token) -e "just testing"
```

File: pkg/cli/root_test.go [hunk 2]
```diff
@@ -28,9 +28,8 @@ Usage: jvsctl COMMAND
   api           Perform API operations
   public-key    Perform public-key operations
   rotation      Perform rotation operations
-  token         Generate a justification token
+  token         Perform token operations
```
Comment (yolocs) [inline pkg/cli/root_test.go:31]: Can you also update https://github.com/abcxyz/jvs/blob/main/prober/prober.sh to reflect the change?

**Review by yolocs — CHANGES_REQUESTED** [review]

File: pkg/cli/root_test.go (hunk 2 above)
Comment (sqin2019) [reply pkg/cli/root_test.go:31]: maybe https://github.com/abcxyz/jvs/blob/main/docs/cli.md#authentication as well?

Comment (dandans-verily) [reply pkg/cli/root_test.go:31]: Updated https://github.com/abcxyz/jvs/blob/main/prober/prober.sh in a new commit. PTAL :D

File: pkg/cli/root.go [hunk 3]
```diff
@@ -69,7 +69,18 @@ var rootCmd = func() cli.Command {
 				}
 			},
 			"token": func() cli.Command {
-				return &TokenCommand{}
+				return &cli.RootCommand{
+					Name:        "token",
+					Description: "Perform token operations",
+					Commands: map[string]cli.CommandFactory{
+						"create": func() cli.Command {
+							return &TokenCommand{}
```
Comment (yolocs) [inline pkg/cli/root.go:77]: nit: Can we now rename this struct to `TokenCreateCommand`?

File: pkg/cli/root.go [hunk 4]
```diff
@@ -69,7 +69,18 @@ var rootCmd = func() cli.Command {
...
+						},
+						"validate": func() cli.Command {
+							return &ValidateCommand{}
```
Comment (yolocs) [inline pkg/cli/root.go:80]: nit: Can we now rename this struct to `TokenValidateCommand`?

**Review by yolocs — APPROVED** [review]

File: pkg/cli/root.go (hunk 3 above)
Comment (dandans-verily) [reply pkg/cli/root.go:77]: Yes, I will rename in a separate followed-up PR so it is easier to see all the other relevant changes.

**Pull request merged**